lib/chromic_pdf/pdfa/ghostscript_worker.ex

```diff
@@ -35,6 +35,15 @@ defmodule ChromicPDF.GhostscriptWorker do
     :ok
   end
 
+  @spec merge(list(binary()), keyword(), binary()) :: :ok
+  def merge(pdf_path_list, _params, output_path) do
+    pdf_path_list = Enum.map(pdf_path_list, &Path.expand/1)
```

Comment:
why is this step necessary?

Reply:
True! Not necessary at all.
It's something I added at the beginning because I was following the example from `convert/3` 😅 
Deleting it 💥